src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -160,12 +153,15 @@ object JsonRpcHttpServer extends Logger {
     }
 
   trait RateLimitConfig {
+    // TODO: Move the rateLimit.enabled setting upwards:
```

Comment:
ditto, maybe we can create another task about it.

Reply:
https://jira.iohk.io/browse/ETCM-509